source/Halibut.Tests/UsageFixture.cs

```diff
@@ -56,6 +56,26 @@ public async Task OctopusCanSendMessagesToTentacle_WithEchoService(ServiceConnec
             }
         }
 
+        [Test]
+        [TestCaseSource(typeof(ServiceConnectionTypesToTest))]
+        public async Task OctopusCanSendMessagesToTentacle_WithEchoService_AndAProxy(ServiceConnectionType serviceConnectionType)
+        {
+            using (var clientAndService = await ClientServiceBuilder
+                       .ForMode(serviceConnectionType)
+                       .WithProxy()
+                       .WithEchoService()
+                       .Build())
+            {
+                var echo = clientAndService.CreateClient<IEchoService>();
+                echo.SayHello("Deploy package A").Should().Be("Deploy package A...");
+
+                for (var i = 0; i < 2000; i++)
```

Comment:
Consider rebasing and using `StandardIterationCount.ForServiceType(serviceConnectionType)` rather than 2k